e2e/e2e_test.go

```diff
@@ -141,26 +141,19 @@ var _ = Describe("e2e", func() {
 
 	Context("with terminating node", func() {
 		BeforeEach(func() {
-			// ensure node is terminating
-			Eventually(func() (bool, error) {
-				if err := k8sClient.Get(context.Background(), ctrl.ObjectKeyFromObject(nodeUnderTest), nodeUnderTest); err != nil {
-					return false, err
-				}
-				for _, cond := range nodeUnderTest.Status.Conditions {
-					if cond.Type == mhc.NodeConditionTerminating {
-						return true, nil
-					}
-				}
-				return false, nil
-			}, 1*time.Minute, 5*time.Second).Should(BeTrue(), "node should not be terminating")
```

Comment:
IIUC we will miss this logic. 
I understand that it is not needed for K8s cluster with kind, but why removing it from OCP as well?

Reply:
It this logic is still relevant then we could move it after line 148 (after we skip when it is a K8s cluster)

Reply:
In the earlier version we marked the node as terminating in the overall BeforeEach block, and here we ensured that it was done. Now we do that here after the k8s check, because otherwise the node would be remediated on k8s. No need to check for it anymore.